server/src/test/java/org/opensearch/extensions/ExtensionsOrchestratorTests.java

```diff
@@ -67,84 +66,79 @@ public void testExtensionsDiscovery() throws Exception {
             "     hostName: 'myIndependentPluginHost1'",
             "     hostAddress: '127.0.0.0'",
             "     port: '9300'",
-            "     version: '3.0.0'",
+            "     version: '1.0'",
```

Comment:
Are we really going to be so boring as to use 1.0 for a version for a test?  There are an infinite number of numbers out there.   0.0.7, for example. 3.14.16.  2.7.18.  42.